damask/bin/run_damask_3.0.0.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+DAMASK_grid -l tensionX.load -g damask.vtr
```

Comment:
```suggestion
mpiexec -n 1 DAMASK_grid -l tensionX.load -g damask.vtr
```

Reply:
Otherwise DAMASK seems to use all cores.

Reply:
If I understood it correctly, DAMASK uses mpi and OMP. So far I do not know which setting is the best for DAMASK. Currently we only support mpi, right?

Reply:
Yes, DAMASK uses both and we only support mpi yet. I can ask DAMASK colleague, which setting they prefer.